mitmproxy.py

```diff
@@ -23,7 +23,16 @@ def binary(self):
 
     def __enter__(self):
         if self.record:
-            command = [self.binary, "--wfile", self.path]
+            command = [
+                self.binary,
+                "--wfile",
+                self.path,
+                "--script",
+                " ".join(
+                    [
+                        os.path.join(self.scripts, "inject-deterministic.py")]
```

Comment:
Does this also need to be used when replaying, or does it just impact the recordings?

Reply:
This only impacts recording. it should not be needed for replay